Seohee/Lv2/124 나라의 숫자.js

```diff
@@ -0,0 +1,10 @@
+function solution(n) {
+    var answer = '';
+    let number124 = [4, 1, 2];
```

Comment:
```suggestion
    let answer = '';
    const number124 = [4, 1, 2];
```
자바스크립트에서 변수를 선언하는 방법이 var, let, const가 있는데 var는 2015년에 es6가 나오면서 let, const가 나오게되면서 치명적인 허점들이 많기때문에 거의 쓰지 않아요. 그래서 보통 원시타입들은 let으로 선언하고 참조타입은 const로 선업합니다.